tests/Php83/Php83Test.php

```diff
@@ -202,4 +202,74 @@ public function testDateTimeExceptionClassesExist()
         $this->assertTrue(class_exists(\DateMalformedIntervalStringException::class));
         $this->assertTrue(class_exists(\DateMalformedPeriodStringException::class));
     }
+
+    /** 
+     * @covers \Symfony\Polyfill\Php83\Php83::str_increment
+     * 
+     * @dataProvider strIncrementProvider
+     */
+    public function testStrIncrement(string $result, string $string)
+    {
+        $this->assertSame($result, str_increment($string));
+    }
+
+    /**
+     * @covers \Symfony\Polyfill\Php83\Php83::str_decrement
+     *
+     * @dataProvider strIncrementProvider
+     */
+    public function testStrDecrements(string $string, string $result)
+    {
+        $this->assertSame($result, str_decrement($string));
+    }
+    
+    public static function strIncrementProvider(): iterable
+    {
+        yield ['ABD', 'ABC'];
+        yield ['EB', 'EA'];
+        yield ['AAA', 'ZZ'];
+        yield ['AAA2', 'AAA1'];
```

Comment:
I suggest adding more tests in those data providers, taking all the cases covered by tests in https://github.com/php/php-src/pull/10358

See https://github.com/php/php-src/pull/10358/files#diff-a892b2d0a9411e1ad34917b112f49c90d4e33f3d61a8491293178d3a5a8fc53b for the tests